engine/lib/plugins.php

```diff
@@ -522,8 +522,6 @@ function _elgg_plugins_init() {
 	elgg_register_action('admin/plugins/deactivate_all', '', 'admin');
 
 	elgg_register_action('admin/plugins/set_priority', '', 'admin');
-
-	elgg_register_library('elgg:markdown', elgg_get_root_path() . 'vendors/markdown/markdown.php');
```

Comment:
Is pointing it to an empty lib just as BC breaking? Why not leave it intact but not use it anymore, and remove it in 2.0?